CodeGeneration/Sources/generate-swift-syntax/templates/swiftsyntax/SyntaxNodesFile.swift

```diff
@@ -22,24 +22,12 @@ import Utils
 func syntaxNode(nodesStartingWith: [Character]) -> SourceFileSyntax {
   SourceFileSyntax(leadingTrivia: copyrightHeader) {
     for node in SYNTAX_NODES.compactMap(\.layoutNode) where nodesStartingWith.contains(node.kind.syntaxType.description.first!) {
-      let documentationSections = [
-        node.documentation,
-        node.grammar,
-        node.containedIn,
-      ]
-      let documentation =
-        documentationSections
-        .filter { !$0.isEmpty }
-        .map { [$0] }
-        .joined(separator: [Trivia.newline, Trivia.docLineComment("///"), Trivia.newline])
-        .reduce(Trivia(), +)
-
       // We are actually handling this node now
       try! StructDeclSyntax(
         """
         // MARK: - \(node.kind.syntaxType)
 
-        \(documentation)
+        \(SwiftSyntax.Trivia(joining: [node.documentation, node.grammar, node.containedIn]))
```

Comment:
@ahoppen I think you told me there's this code in _three_ spots, but I only caught two: here, and in `Child.swift`. Where's the third one?

Reply:
There’s another one in SyntaxCollectionsFile.swift:21